src/app/shell/Destiny.tsx

```diff
@@ -167,19 +172,25 @@ export default function Destiny() {
   });
   useHotkeys(hotkeys);
 
+  if (
+    !accountsLoaded ||
+    // This delays to wait for current account to be set in Redux so we don't get ahead of ourselves
+    (account && !currentAccount)
```

Comment:
This page computes the current account based on path, not what's in Redux, so this line just delays the full render of child pages until Redux has caught up.